core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -473,7 +473,13 @@ class ZkUtils(val zkClient: ZkClient,
   }
 
   def createSequentialPersistentPath(path: String, data: String = "", acls: java.util.List[ACL] = DefaultAcls): String = {
-    ZkPath.createPersistentSequential(zkClient, path, data, acls)
+    try {
```

Comment:
Do we still need this fix? Is it something we should submit upstream outside of EoS?